lib/internal/cluster/master.js

```diff
@@ -49,7 +49,8 @@ cluster.setupMaster = function(options) {
     silent: false
   };
   settings = util._extend(settings, cluster.settings);
-  settings = util._extend(settings, options || {});
+  if (options)
```

Comment:
I think we should code for clarity, not performance, on one-shot setup functions. The clearest version of this line is `settings = util._extend(settings, options)`, because it doesn't have any extra code. If you  can simply delete lines from a file, and it still does exactly what it did before, its hard to understand for a reader why the line is there in the first place.